WebFileSystem.ts

```diff
@@ -635,7 +635,7 @@ class WebFileSystem extends webdav.FileSystem {
                 callback(webdav.Errors.None)
             }
         } else {
-            logger.error(webdav.Errors.BadAuthentication.message)
+            logger.error(`WebFileSystem._size.user.false : ${webdav.Errors.BadAuthentication.message}`, new Error('Stack-Tracer'))
```

Comment:
Error vs. Warning (there are a few more of this cases, I am not going to mention every single one)